scripts/setup.sh

```diff
@@ -8,9 +8,9 @@ export AUTHJS_VERSION=""
 # Install required node version
 export REGISTRY_REPO="npm-topic"
 export REGISTRY="${ARTIFACTORY_URL}/api/npm/${REGISTRY_REPO}"
-setup_service node v12.13.0
+setup_service node v12.22.12
 # Use the cacert bundled with centos as okta root CA is self-signed and cause issues downloading from yarn
-setup_service yarn 1.21.1 /etc/pki/tls/certs/ca-bundle.crt
+setup_service yarn 1.18.0 /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
Is this issue with yarn (which I understand is related to adding new packages) relevant to Bacon?

Reply:
Yes. Specifically it's an issue with `yarn workspace`

https://github.com/yarnpkg/yarn/issues/7807

Reply:
Not related to Bacon in particular